devtools/travis-ci/install.sh

```diff
@@ -20,6 +20,8 @@ conda update --yes conda
 conda install --yes conda-build jinja2 anaconda-client pip
 conda install --yes -c omnia openmoltools
 conda install --yes -c omnia parmed
+conda install --yes -c matplotlib
+conda install --yes pandas=0.13.1
```

Comment:
@bannanc - maybe the attempt to ask for pandas=0.13.1 is causing problems? I'm getting an "unsatisfiable dependencies" issue on my laptop if I do it this way, whereas just asking for pandas works fine. (I get 0.19.2).